public/js/project/game2/level2.js

```diff
@@ -1,10 +1,9 @@
 var Canvas = require('../canvas/canvas.js'),
 		Vector = require('../actors/vector.js'),
 		Target = require('../actors/target.js'),
-		Sauron = require('../sauron/sauron.js'),
-		config = require('./config.js'),
-		Level2 = new Sauron(config.sauron);
-		console.log(Level2.level)
+		Sauron = require('../sauron/sauron.js')
+		config = require('./config.js');
+		OverWatcher = new Sauron(config.sauron);
```

Comment:
Same comment for `level 1`.
Scope should be fine here.
No need to make it global.